apps/deploy-web/src/pages/user/verify-email/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { buttonVariants } from "@akashnetwork/ui/components";
+import { cn } from "@akashnetwork/ui/utils";
+import { ArrowRight } from "iconoir-react";
+import Link from "next/link";
+import { useSearchParams } from "next/navigation";
+import { NextSeo } from "next-seo";
+
+import Layout from "@src/components/layout/Layout";
+import { Title } from "@src/components/shared/Title";
+import { useWhen } from "@src/hooks/useWhen";
+import { useVerifyEmail } from "@src/queries/useVerifyEmailQuery";
+import { UrlService } from "@src/utils/urlUtils";
+
+export default function VerifyEmailPage() {
+  const email = useSearchParams().get("email");
+  const [isVerified, setIsVerified] = useState<boolean | null>(null);
+  const { mutate: verifyEmail, isPending: isVerifying } = useVerifyEmail({ onSuccess: setIsVerified });
+
+  useWhen(email, () => {
+    if (email) {
+      verifyEmail(email);
+    }
+  });
+
+  return (
+    <Layout isLoading={isVerifying}>
+      <NextSeo title="Verifying your email" />
+
+      <div className="mt-10 text-center">
```

Comment:
We can add the Akash Console logo and big icon (check or X or spinner) to make it prettier and more interactive.